src/parameterizations/lateral/MOM_self_attr_load.F90

```diff
@@ -193,7 +193,7 @@ subroutine SAL_init(G, GV, US, param_file, CS)
   logical :: calculate_sal=.false.
   logical :: tides=.false., use_tidal_sal_file=.false., bq_sal_tides_bug=.false.
   integer :: tides_answer_date=99991231 ! Recover old answers with tides
-  real :: sal_scalar_value, tide_sal_scalar_value ! Scaling SAL factors [nondim]
+  real :: sal_scalar_value=0.0 ! Scaling SAL factors [nondim]
```

Comment:
I don't think that there is any need to initialize `sal_scalar_value` in this case.  Also, the MOM6 convention is that we use spaces around the ` = ` when setting a parameter (as opposed to giving the name of an optional argument), so if it is retained, this line should be `sal_scalar_value = 0.0`.

Reply:
Done